IRONIC.md

```diff
@@ -33,3 +109,34 @@ following variables in `` user_osa_variables_overrides.yml ``:
           dhcp-option-force: "26,1500"
           dhcp-ignore: "tag:!known"
           log-facility: "/var/log/neutron/dnsmasq.log"
+
+## Post deployment setup of neturon
+
+Ironic / OpenStack needs to be deployed before you can finish the configuration
+of Ironic.  Once the initial deployment is done, you can proceed.
+
+### Set up the neutron network and configure the cleaning network
+
+	neutron net-create --shared --provider:physical_network tftp --provider:network_type flat tftp
+	neutron subnet-create --name ironic-tftp \
+	  --allocation-pool start=<NON_ALLOCATED_TFTP_RANGE_START>,end=<NON_ALLOCATED_TFTP_RANGE_END> \
+	  --dns-nameserver=4.4.4.4 tftp <TFTP_NETWORK>/<TFTP_NETMASK>
+
+Once the network is created, take the output of 
+`` neutron net-list | grep tftp | awk '{print $2}' `` and add the following
+section to your `` user_osa_variables_overrides.yml ``.
+
+	ironic_ironic_conf_overrides:
+	  neutron:
+	    cleaning_network_uuid: "<NEUTRON_TFTP_NETWORK_UUID>"
+	  conductor:
+	    automated_clean: true
+
+This will autoclean the nodes on checkin and when they are initially registered
+to Ironic.
+
+### Redeploy Ironic
+
+Rerun the `` os-ironic-install.yml `` play to get the new settings in place.
+
+## todo: Post deployment image registration and node enrollment
```

Comment:
are we expecting this in a follow on pr or are you going to continue adding to this one?